csv-data-management/extract_data_from_csv.py

```diff
@@ -200,6 +200,119 @@ def merge_two_csv_and_extract_data(config, params):
         logger.error('Error in extract_data_from_csv(): %s' % Err)
 
 
+def concat_two_csv_and_extract_data(config, params):
+    try:
+        numberOfRowsToSkipFirst = None
+        numberOfRowsToSkipSecond = None
+        isSingleColumnFrist = None
+        isSingleColumnSecond = None
+        isCSVWithoutHeadersFirst = False
+        isCSVWithoutHeadersSecond = False
+
+        file_1_iri_ = handle_params(params,params.get('file_one_value'))
+        file_2_iri_ = handle_params(params,params.get('file_two_value'))
+        logger.warning(params)
+        first_file_path = join('/tmp', download_file_from_cyops(file_1_iri_)['cyops_file_path'])
+        second_file_path = join('/tmp', download_file_from_cyops(file_2_iri_)['cyops_file_path'])
+
+        res = _check_if_csv(first_file_path)
+        logger.info(res)
+  
+        if res.get('headers') == False:
+            isCSVWithoutHeadersFirst = True
+
+        if res.get('columns') == 1:
+            isSingleColumnFrist = True
+        
+        res = _check_if_csv(second_file_path)
+
+        if res.get('headers') == False:
+            isCSVWithoutHeadersSecond = True
+        elif res.get('columns') == 1:
+            isSingleColumnSecond = True
+
+
+        #Lets read first file 
```

Comment:
Can be converted into a small func and called for both files one by one

Reply:
Will take it up in next iteration , will require some code layout change